src/test/java/org/folio/qm/controller/KafkaListenerApiTest.java

```diff
@@ -18,7 +18,7 @@
 
 class KafkaListenerApiTest extends BaseApiTest {
 
-  private static final UUID JOB_EXECUTION_ID = UUID.fromString("a7fb1c32-1ffb-4a22-a76a-4067284fe68d");
+  static final UUID JOB_EXECUTION_ID = UUID.fromString("a7fb1c32-1ffb-4a22-a76a-4067284fe68d");
```

Comment:
Maybe will be better to move it to `TestUtils` if it is needed in other test classes